delegation-verify-coordinator/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "delegation-verify-coordinator.fullname" . }}
+  labels:
+    {{- include "delegation-verify-coordinator.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "delegation-verify-coordinator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "delegation-verify-coordinator.labels" . | nindent 8 }}
+	      {{- with .Values.podLabels }}
```

Comment:
this might need a touch ^^

Reply:
@Smorci, it looks that you missed this one.